test/cmds/watch/watch.test.js

```diff
@@ -0,0 +1,173 @@
+const EventEmitter = require('events');
+const chokidar = require('chokidar');
+const watch = require('../../../dist/src/cmds/watch');
+const build = require('../../../dist/src/cmds/build/bundle');
+const fsUtils = require('../../../dist/src/utils/validate-fs');
+const miscUtils = require('../../../dist/src/utils/misc');
+
+describe('watch', () => {
+  describe('Watch a directory and its subdirectories for changes, and build when files are added or changed.', () => {
+    let watcherEmitter;
+
+    const mockSrc = 'index.js';
+    const mockDist = 'dist';
+    const mockOutfileName = 'bundle.js';
+
+    const getMockArgv = () => {
+      return {
+        src: mockSrc,
+        dist: mockDist,
+        outfileName: mockOutfileName,
+      };
+    };
```

Comment:
Okay, I did this instead. When we need to check a particular property, we check the appropriate mock constant instead.